src/api/helpers/create-a-declaration-modern-slavery/index.test.ts

```diff
@@ -0,0 +1,80 @@
+import createADeclarationModernSlavery from '.';
+import { mockInvalidId } from '../../test-mocks';
+import createADeclarationModernSlaveryVersion from '../create-a-declaration-modern-slavery-version';
+import getKeystoneContext from '../../test-helpers/get-keystone-context';
+import applications from '../../test-helpers/applications';
+import declarations from '../../test-helpers/declarations';
+import { Application, ApplicationDeclaration, Context } from '../../types';
+
+describe('helpers/create-a-declaration-modern-slavery', () => {
+  let context: Context;
+  let application: Application;
+  let declaration: ApplicationDeclaration;
+
+  beforeAll(async () => {
+    context = getKeystoneContext();
+
+    application = (await applications.create({ context })) as Application;
+    declaration = (await declarations.create(context)) as ApplicationDeclaration;
+  });
+
+  test('it should return a declaration modern slavery with ID', async () => {
+    const result = await createADeclarationModernSlavery(context, declaration.id);
+
+    expect(result.id).toBeDefined();
+    expect(typeof result.id).toEqual('string');
+    expect(result.id.length).toBeGreaterThan(0);
+  });
+
+  test('it should return a declaration ID', async () => {
```

Comment:
I just realised.  Should these not be its? and should start with should :P

Reply:
We have over 300 instances of this, I believe from the start of the project. Can tackle this in another PR.